File: WICI2_Documentation/tags/Mar_04_Release/WICI2MobileWebArtifacts/debug/app/controller/models/ProvincesList.js
ensureNamespaceExists();

//luk: old name: WICIWebApp.store.Provinces
//list of all possible provinces

WICI.ProvincesList = function() {
    this.data= [
          { value: null, text: 'ProvincesList_null'},
          { value: 'AB', text: 'ALBERTA'},
          { value: 'BC', text: 'BRITISH COLUMBIA'},
          { value: 'MB', text: 'MANITOBA'},
          { value: 'NB', text: 'NEW BRUNSWICK'},
          { value: 'NL', text: 'NEWFOUNDLAND & LABRADOR'},
          { value: 'NS', text: 'NOVA SCOTIA'},
          { value: 'NT', text: 'NORTHWEST TERRITORIES'},
          { value: 'NU', text: 'NUNAVUT'},
          { value: 'ON', text: 'ONTARIO'},
          { value: 'PE', text: 'PRINCE EDWARD ISLAND'},
          { value: 'QC', text: 'QUEBEC'},
          { value: 'SK', text: 'SASKATCHEWAN'},
          { value: 'YT', text: 'YUKON'}
      ];
};

File: BRB_Documentation/tags/02_25_2014_BRB_PRC_v1/BRBWeb_Test/src/main/java/com/epam/brb/TestData/ProvincesList.java
package com.epam.brb.TestData;

import java.util.ArrayList;
import java.util.Iterator;
import java.util.List;

import org.testng.annotations.DataProvider;

public class ProvincesList {

		@DataProvider(name = "ProvincesList", parallel=false)
	    public static Iterator<Object[]> createData() {
	        List<Object[]> data = new ArrayList<Object[]>();
	        data.add(new Object[]{"ALBERTA"});
	        data.add(new Object[]{"BRITISH COLUMBIA"});
	        data.add(new Object[]{"MANITOBA"});
	        data.add(new Object[]{"NEW BRUNSWICK"});
	        data.add(new Object[]{"NEWFOUNDLAND & LABRADOR"});
	        data.add(new Object[]{"NOVA SCOTIA"});
	        data.add(new Object[]{"NORTHWEST TERRITORIES"});
	        data.add(new Object[]{"NUNAVUT"});
	        data.add(new Object[]{"ONTARIO"});
	        data.add(new Object[]{"PRINCE EDWARD ISLAND"});
	        data.add(new Object[]{"QUEBEC"});
	        data.add(new Object[]{"SASKATCHEWAN"});
	        data.add(new Object[]{"YUKON"});
	        
	        return data.iterator();
	    }
	}


